### PR TITLE
Add work-out option

### PR DESCRIPTION
**Pull request by ME97**

### CODE REVIEW

**Comment by ME97** [comment]
@casey-collab

File: english/work-out/work-out.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Now that you've made a good life desicion and decided to work out, you now must
+choose your method. Do you:
+
+Work that cardiovascular system by going for a [run](/run/run.md)?
```
Comment (casey-collab) [inline english/work-out/work-out.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/work-out/work-out.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+Or...
+
+Attempt to get super swole by lifting some [weights](/weights/weights.md)?
```
Comment (casey-collab) [inline english/work-out/work-out.md]: This link is also invalid.

File: english/work-out/weights/weights.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+You head to the garage and dust off the old bench press. You load up 135 pounds,
```
Comment (casey-collab) [inline english/work-out/weights/weights.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/work-out/run/run.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+You lace up your 10 year old tennis shoes and hit the road running.
```
Comment (casey-collab) [inline english/work-out/run/run.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ME97** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :palm_tree:

**Pull request merged**